scooby/report.py

```diff
@@ -297,7 +297,7 @@ def cols(html, version, name, ncol, i):
         # Loop over packages
         i = 0  # Reset count for rows.
         for name, version in self.packages.items():
-            html, i = cols(html, version, name, self.ncol, i)
+            html, i = cols(html, name, version, self.ncol, i)
```

Comment:
Actually we don't want to do any of these changes but switch where the variables are used **inside** the `cols` method. These changes pass the `name` as the `version` argument and `version` as the `name` argument leading to a significant inconsistency. Revert this change and change the internals of the `cols` method to the following:

```py
def cols(html, version, name, ncol, i):
            r"""Print package information in two cells."""

            # Check if we have to start a new row
            if i > 0 and i % ncol == 0:
                html += "  </tr>\n"
                html += "  <tr>\n"

            html += "    <td style='text-align: right; background-color: #ccc;"
            html += " " + border + ">%s</td>\n" % name

            html += "    <td style='text-align: left; "
            html += border + ">%s</td>\n" % version

            return html, i+1
```

Reply:
Ok. I'll do that. 

Another question. At the moment I left the space for RAM empty. Should I write RAM inside and write an ? or _ or something else as value? 

![image](https://user-images.githubusercontent.com/3840578/74076802-f51ac980-4a1a-11ea-930f-95cc923cf3a6.png)

If I leave it as before, the edge is missing a border, like here:

![image](https://user-images.githubusercontent.com/3840578/74076886-b33e5300-4a1b-11ea-94fe-1381dc4e715f.png)

Reply:
Don't mind the yellow spot.

Reply:
can you install `psutil` so Scooby will report on the RAM? It's an optional dependency

Reply:
That edge is weird, though... not sure why that is happening